v1/0_introduction.md

```diff
@@ -0,0 +1,27 @@
+# Introduction to Coinswap
+
+Coinswap is a protocol in Bitcoin that enables trustless, decentralized coin swaps between parties without requiring intermediaries. Originally proposed by Greg Maxwell in 2013 on bitcointalk.org, the protocol has evolved significantly, with Chris Belcher introducing important improvements in 2020 through the bitcoin-dev mailing list.
```

Comment:
Exapnd on the Chris Belcher part a bit more.

- He gave a complete design covering the edge cases. In the mailing lists. https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2020-October/018221.html
- a more detailed design doc mentioning future improvements. https://gist.github.com/chris-belcher/9144bd57a91c194e332fb5ca371d0964
- Then he built a Proof of Concept in rust. https://github.com/bitcoin-teleport/teleport-transactions
- He couldn't continue further due to health issues. The project got stale on June 2021. https://gist.github.com/chris-belcher/ca5051285c6f8d38693fd127575be44d
- We picked up where he left and carried on the building. This is still visible in our main repo as "forked from" tag. https://github.com/citadel-tech/coinswap 

This project is a continuation of belcher's legacy. We will finish what he started.